src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
Right, and that's the point of the `<=`... we don't know when it was introduced, but it exists as of a known version (in this case, 9), which we don't need to update when our browser support changes.